app/controllers/concerns/recaptchable.rb

```diff
@@ -0,0 +1,24 @@
+module Recaptchable
+  extend ActiveSupport::Concern
+
+  included do
+    def verify_recaptchas
+      v2_secret_key = Rails.application.credentials.recaptcha_v2[:secret_key]
+      v3_secret_key = Rails.application.credentials.recaptcha_v3[:secret_key]
+
+      success = verify_recaptcha(
+        action: params[:action],
+        minimum_score: 0.5,
+        secret_key: v3_secret_key
+      )
+
+      unless success
+        checkbox_success = verify_recaptcha(secret_key: v2_secret_key)
+      end
+
+      @show_checkbox_recaptcha = !success
```

Comment:
maybe rename to something like @show_v2_recaptcha ? just a suggestion

Reply:
https://developers.google.com/recaptcha/docs/v3

Scroll down to "Interpreting the score"

Reply:
Oh shit, this was for the other question.

> 0.5 comes from what/where?

Reply:
> maybe rename to something like @show_v2_recaptcha ? just a suggestion

yes that works